core.md

```diff
@@ -397,40 +463,27 @@ functions (but see section on opacity below):
 
 ## Opacity
 
-Sometimes it is useful to be able to prevent others from gaining
-access to the buffer in which a typed object is stored. This can
-provide a measure of protection, since otherwise passing a pointer to
-(say) an individual array element also provides access to the entire
-array.
-
-To this end, any typed object can be converted into an opaque typed
-object using the `opaque` function. This function returns a new typed
-object that points at the same buffer but is opaque (the input object
-is not modified and remains transparent). Opaque typed objects always
-return `undefined` for the `buffer`, `offset`, and `length` funtions.
-
-    var point = new Point();
-    var point1 = opaque(point);
-    
-    buffer(point)  // yields a buffer
-    offset(point)  // yields 0
-    length(point)  // yields 16
-    
-    buffer(point1) // yields undefined
-    offset(point1) // yields undefined
-    length(point1) // yields undefined
-
-### Opaque types
-
-For types that contain object or string pointers, opacity is more than
-a convenience, it's a security necessity. If users could gain access
-to the backing buffer, then they could synthesize fake pointers and
-hack the system. Therefore, type definitions that contain pointers are
-classified as *opaque type definitions*. All instances of an opaque
-type definition are automatically opaque, and the `view` method on an
-opaque type definition throws an exception.
-
-*NOTE:*
-[Issue #2](https://github.com/nikomatsakis/typed-objects-explainer/issues/2)
-proposes to change some details of this section.
+By default, struct types are opaque, meaning they don't allow access to
+the buffer onto which the struct instance is a view. This is a measure of
+protection, since otherwise passing a pointer to (say) an individual nested
+struct also provides access to the entire outer struct.
+
+Sometimes, though, it's useful to allow accessing the underlying buffer. This can be enabled on a per-type basis using the `transparent` option:
+
+```js
+const TransparentPoint = new StructType({x: float64, y: float64}, {transparent: true});
+const Point = new StructType({x: float64, y: float64});
+let point = new TransparentPoint({x: 10, y: 100});
+let opaquePoint = Point.view(buffer(point), offset(point));
```

Comment:
I don't think we want to allow view with an opaque type. Besides th e security concerns, it will impair optimization, since we no longer have aliasing guarantees.